src/PaymentGateways/Gateways/Stripe/StripePaymentElementGateway/resources/js/StripeAccountPanel.tsx

```diff
@@ -0,0 +1,108 @@
+import {__} from '@wordpress/i18n';
+import {InspectorControls} from '@wordpress/block-editor';
+import {Notice, PanelBody, SelectControl, ToggleControl} from '@wordpress/components';
+import {createInterpolateElement} from '@wordpress/element';
+import './styles.scss';
+
+export default function StripeAccountPanel({
+    attributes,
+    setAttributes,
+}: {
+    attributes: StripeProps;
+    setAttributes: ({}) => void;
+}) {
+    // @ts-ignore
+    const stripeGlobalSettings: StripeGlobalSettingsProps = window.stripePaymentElementGatewaySettings;
```

Comment:
Here's a little trick i've been using to scope window data.

Add this before the component and then you can remove `@ts-ignore`

```
declare const window: {
    stripePaymentElementGatewaySettings: StripeGlobalSettingsProps;
} & Window;
```

This is my recommended approach for defining window data because it's a single instance of the window and not a global - which has caused problems with our multi-domain application structure.

If you peek into the form builder getWindowData file you'll see being used.

Reply:
Resolved: a0285f5db0012bc474732e2f374619dc3045c2a7